### PR TITLE
Improve documentation around configuring specific fixers

### PR DESCRIPTION
**Pull request by spawnia**
Thanks for this great service.

I am always having a hard time when i am writing a StyleCI configuration file.

When i visit https://docs.styleci.io/fixers to look for a specific option, it does not specify where and how i can actually enable that option. The only actual code example of enabling/disabling a fixer is found on https://docs.styleci.io/multi-language

This PR adds a short introductory paragraph that shows a quick example of how fixers can be configured.

Also, i edited the configuration page to give more succinct, focused information that is directed towards getting a working configuration up and running.

### CODE REVIEW

File: fixers.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # PHP Fixers
 
-We have many available fixers. Note that while header comment fixing is available in StyleCI, we do not consider it a "fixer" as isn't configured as a fixer. Header comment fixing has it's own dedicated config on the settings page for your repo.
+Our PHP fixing works by applying single-purposed transformations to files that fix specific issues.
+
+Depending upon the [preset](presets), a default set of fixers will be included.
+If you want more granular control, you can enable or disable specific fixers by adding them to your configuration:
+
+```diff
```
Comment (GrahamCampbell) [inline fixers.md]: The format should is `yaml` here. ;)

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: fixers.md (hunk 1 above)
Comment (spawnia) [reply fixers.md]: I chose `diff` to show adding/removal of the lines. We can go with plain `yaml` too if you like that better.

Comment (GrahamCampbell) [reply fixers.md]: I'm not sure how this will actually render on our docs site. I'll try it out in the next few days, locally.

Comment (spawnia) [reply fixers.md]: Should i just change this to `yaml` so this PR can proceed?

Comment (spawnia) [reply fixers.md]: Changed it.

**Comment by spawnia** [comment]
@GrahamCampbell did your requested change, anything else?

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Pull request merged**